transmart-server/grails-app/conf/logback.groovy

```diff
@@ -0,0 +1,39 @@
+import grails.util.BuildSettings
```

Comment:
ok, I changed it as you suggested (although github not marking this as outdated)